backend/geton_rpg/urls.py

```diff
@@ -12,7 +12,7 @@
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('api/', include(('core.urls'), namespace='core')),
-    path('api/', include(('users.urls'), namespace='users')),
+    path('', include(('users.urls'), namespace='users')),
```

Comment:
Why not add the users in the api url?

Reply:
```suggestion
    path('', include(('users.urls'), namespace='users')),
```
Because, the new(s) route as writing in users/routes.py